.github/workflows/build.yml

```diff
@@ -71,13 +71,8 @@ jobs:
     env:
       ELECTRON_VERSION: ${{ matrix.electron }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: volta-cli/action@v3
-      - uses: actions/cache@v2
```

Comment:
This caches downloaded Electron binary files. This won't be much faster than just downloading them again while using up a lot of the cache allowance